src/System.Runtime.Extensions/tests/System/IO/PathTests_Windows.netcoreapp.cs

```diff
@@ -33,5 +33,15 @@ public void GetFullPath_TrailingPeriodsCut(string component)
             string path = "C:\\Test" + component;
             Assert.Equal("C:\\Test", Path.GetFullPath(path));
         }
+
+        [Theory,
+           MemberData(nameof(GetFullPath_Windows_NonFullyQualified)),
+           MemberData(nameof(GetFullPath_Windows_FullyQualified)),
+           MemberData(nameof(GetFullPath_Windows_UNC))]
```

Comment:
@JeremyKuhne these tests are not getting run even after restructuring

Reply:
While @jkotas fixed #27426, we still need to ingest the updated bits for the problem to go away. Personally I've been running "build -windowsmscorlib" in CoreCLR and copying System.Private.CoreLib to the "\bin\runtime" folder in CoreFX to run tests to unblock myself.